GoogleSignIn/Sources/GIDConfiguration.m

```diff
@@ -26,6 +26,12 @@
 // The key for the openIDRealm property to be used with NSSecureCoding.
 static NSString *const kOpenIDRealmKey = @"openIDRealm";
 
+// Info.plist config keys
+static NSString *const kConfigClientIDKey = @"GIDClientID";
```

Comment:
I think these are also defined on [GIDSignIn.m](https://github.com/google/GoogleSignIn-iOS/blob/ec20caf39917e3a50905776db1713b37c6b33840/GoogleSignIn/Sources/GIDSignIn.m#L139), right? Perhaps we can make a constant types so that we don't have duplicate code?

Reply:
Or, if they're no longer used in `GIDSignIn`, then maybe just remove them from there?

Reply:
Removed since they're no longer used in `GIDSignIn`.